src/test/resources/pages/instructorStudentRecordsPageEditedComment.html

```diff
@@ -0,0 +1,701 @@
+                                        </td>
+                                      </tr>
+                                    </tbody>
+                                  </table>
+                                </div>
+                                <div class="form-group">
+                                  <div class="panel panel-default panel-body content-editor" id="responsecommenttext-RGQ-1-1-1-1" spellcheck="false" style="position: relative;">
+                                    <p>
+                                      First edited comment to Alice about feedback to Benny from different Instructor
+                                    </p>
+                                  </div>
+                                  <input name="responsecommenttext" type="hidden" value="<p>First edited comment to Alice about feedback to Benny from different Instructor</p>">
+                                </div>
+                                <div class="col-sm-offset-5">
+                                  <a class="btn btn-primary" href="/page/instructorFeedbackResponseCommentEdit" id="button_save_comment_for_edit-RGQ-1-1-1-1" type="button">
+                                    Save
+                                  </a>
+                                  <input class="btn btn-default hide-frc-edit-form" data-frcindex="1" data-giverindex="1" data-qnindex="1" data-recipientindex="1" data-viewtype="RGQ" type="button" value="Cancel">
+                                </div>
+                                <input name="responsecommentid" type="hidden" value="${comment.id}">
+                                <input name="fsindex" type="hidden" value="1">
+                                <input name="responseid" type="hidden" value="${question.id}%alice.b.tmms@gmail.tmt%benny.c.tmms@gmail.tmt">
+                                <input name="courseid" type="hidden" value="ISR.CS2104">
+                                <input name="fsname" type="hidden" value="First feedback session">
+                                <input name="user" type="hidden" value="ISR.teammates.test">
+                                <input name="showresponsecommentsto" type="hidden" value="INSTRUCTORS">
+                                <input name="showresponsegiverto" type="hidden" value="INSTRUCTORS">
+                                <input name="token" type="hidden" value="${sessionToken}">
+                              </form>
+                            </li>
+                            <li class="list-group-item list-group-item-warning" id="responseCommentRow-RGQ-1-1-1-2">
+                              <div class="row" id="commentBar-RGQ-1-1-1-2" style="display: block;">
+                                <div class="col-xs-10">
+                                  <span class="text-muted">
+                                    From: Teammates Test [Mon, 02 Mar 2026, 01:59 AM UTC+0200] (last edited by Teammates Test at ${datetime.now} UTC+0200)
```

Comment:
No, the problem is with testing. Every time you create a comment or edit an existing one new timestamp will be generated for the comment. Suppose you run the test on your dev server with GodMode `on`, HTML files will have a timestamp of when you did the testing. Now when you run the Travis build, it will have a timestamp of that moment different than the timestamp of your HTML file and thus, will throw an assertion error. So to avoid that we replace that set of unpredictable values using function `replaceUnpredictableValuesWithPlaceholders` in `HtmlHelper.java`.